pontoon/homepage/views.py

```diff
@@ -28,6 +30,23 @@ def homepage(request):
     else:
         start_url = reverse('pontoon.teams')
 
-    return render(request, 'homepage.html', {
+    playground_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'tutorial',
+        'part': 'playground',
+    })
+
+    intro_url = reverse('pontoon.translate.locale.agnostic', kwargs={
+        'slug': 'pontoon-intro',
+        'part': 'messages.properties',
+    })
```

Comment:
Why did you move these out of the template? I think it's fine to keep them there, and use the `url` helper to construct the URLs.

I guess you followed the `start_url` pattern, but I think that one is an exception, because we want to be smart with it. The other 2 don't change depending on the context, so they should stay in the template in my opinion.

Reply:
I was facing problems in rendering the template using template.render() . It was throwing a parse error while using url() or static() words in template.

Reply:
I think in the case of a customizable homepage it's actually OK to move the logic out of the template, so that editing will be easier. I'd rename `intro_url` to `in_context_url`, though.

Reply:
Not being able to create links in the template might be a problem in the future. It might be worth trying to figure out what that is so, and fix it.